### PR TITLE
Implement #570 #571 #572 high-priority runtime slices

### LINKED ISSUE #570: task: unblock sync-latency harness measurements for summary actions
## Context
`docs/STATUS.md` identifies a concrete blocker for clean sync-latency measurement in follow-up issue #433: the watcher allowlist in `vault/@Settings/watchers.md` only allows `promote.evergreen`, while `ingest.summary.create` used by summary-type test notes is known but not allowlisted. This causes zero-measurement runs despite validated iCloud transport.

## Scope
Unblock deterministic sync-latency measurement for #433:
- update watcher allowlist/settings handling so `ingest.summary.create` can be enabled for harness runs
- keep default production-safe posture intact
- add harness + watcher tests proving non-zero measurement path when summary actions are allowed
- document the operator-safe toggle path for measurement mode

## Source Anchors
- `docs/STATUS.md` :: Sync-latency harness — partial acceptance (2026-04-12) blocker note
- `docs/STATUS.md` :: Follow-up timing receipt tracked in #433
- `docs/ROADMAP.md` :: Closed v5.6 line and post-delivery sync follow-up framing

## Constraints
- Preserve conservative default allowlist behavior for normal runtime.
- Restrict broadened allowlist behavior to explicit harness/operator mode.
- Keep watcher write-safety and provenance checks unchanged.
- Avoid adding non-deterministic dependencies to the harness path.

## Acceptance Criteria
- [ ] Harness/operator mode can include `ingest.summary.create` in watcher allowlist without changing default runtime allowlist.  
  Verify: `tests/settings/test_panel_watcher_config_validation.py::test_summary_action_allowlisted_only_in_measurement_mode`.
- [ ] Sync-latency harness records at least one measurement on summary-action notes when measurement mode is enabled.  
  Verify: `tests/cli/test_sync_latency_harness.py::test_harness_records_measurement_when_summary_action_allowed`.
- [ ] Status/runbook docs describe the safe operator path for enabling measurement mode and returning to default posture.  
  Verify: doc writeback at `docs/STATUS.md` and/or `docs/OPERATIONS.md` with measurement-mode note.
- [ ] #433 can reference this issue as the unblocker for clean measurement capture.  
  Verify: issue linkage note in `#433` comment after merge.

## Out of Scope
- Mac mini headless infrastructure hardening work tracked separately.
- Reworking sync transport chain instrumentation.
- Any v6 capability expansion.

## Suggested Validation
- `PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 pytest -q tests/settings/test_panel_watcher_config_validation.py -k summary_action`
- `PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 pytest -q tests/cli/test_sync_latency_harness.py`
- `python -m app.cli sync-latency-harness --help`

## Source Docs
- `docs/STATUS.md`
- `docs/ROADMAP.md`
- `docs/OPERATIONS.md`

### LINKED ISSUE #571: task: add derived salience/staleness signal seam for v6 cognitive support
## Context
`docs/plans/V60_COGNITIVE_SUPPORT_PRIORITIES.md` identifies Priority 1 (salience + staleness signals) as the first unblocker for cognitive-support surfaces. Existing retrieval metadata work (#539) and stale/partial view signaling (#537) landed targeted increments, but there is no bounded issue that establishes a unified runtime salience/staleness signal interface consumable by resurfacing/orientation follow-ups.

## Scope
Implement a minimal runtime signal interface for salience/staleness derivation:
- define a small typed signal payload used at decision time (not persisted as artifact truth)
- compute baseline inputs from existing runtime evidence (recent interaction, open-loop pressure proxies, temporal drift markers)
- expose the signal payload through a non-authoritative capability seam for future orientation/resurfacing consumers
- keep all outputs marked as derived/ephemeral and avoid schema changes that store salience as durable artifact state

## Source Anchors
- `docs/plans/V60_COGNITIVE_SUPPORT_PRIORITIES.md` :: Priority 1 — Operationalize salience and staleness as first-class signals
- `docs/CONCEPTS/SALIENCE_AND_ATTENTIONAL_RELEVANCE_CONTRACT.md`
- `docs/CONCEPTS/TEMPORAL_VALIDITY_AND_STALENESS_CONTRACT.md`
- `docs/FINDING_AND_REORIENTING/DOCUMENT_SALIENCE_AS_DERIVED.md`

## Constraints
- No durable `salience` field on stored artifacts.
- No ranking-policy changes in retrieval scoring in this slice.
- No user-facing UI changes; capability seam only.
- Preserve backward compatibility for existing retrieval/status APIs.

## Acceptance Criteria
- [ ] Runtime exposes a typed derived salience/staleness signal payload through a capability seam without persisting it to artifact records.  
  Verify: `tests/retrieval/test_salience_signals.py::test_salience_signals_are_derived_not_persisted`.
- [ ] Staleness derivation remains distinct from `maturity` and `review_state` semantics.  
  Verify: `tests/retrieval/test_salience_signals.py::test_staleness_not_sourced_from_state_axes_labels`.
- [ ] Consumers can request the signal payload without mutating existing retrieval contract outputs by default.  
  Verify: `tests/retrieval/test_retrieval_capability.py::test_retrieval_signal_payload_opt_in`.
- [ ] Docs record the new seam and derived-only guardrail.  
  Verify: doc writeback at `docs/RETRIEVAL.md` and `docs/STATUS.md`.

## Out of Scope
- Orientation/resurfacing product surfaces.
- Commitment runtime implementation.
- Full salience model tuning.

## Suggested Validation
- `PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 pytest -q tests/retrieval/test_salience_signals.py`
- `PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 pytest -q tests/retrieval/test_retrieval_capability.py -k signal`

## Source Docs
- `docs/plans/V60_COGNITIVE_SUPPORT_PRIORITIES.md`
- `docs/FINDING_AND_REORIENTING/README.md`
- `docs/RETRIEVAL.md`

### LINKED ISSUE #572: task: enforce Panel trust-verb gating and receipt fields for APPLY/SUGGEST
## Context
`docs/plans/V60_COGNITIVE_SUPPORT_PRIORITIES.md` Priority 3 requires receipt-bearing SUGGEST/APPLY gating before widening Chat or broader automation authority. Promotion transition receipts (#535) are shipped, but there is no bounded issue establishing verb-level receipt fields and gate behavior at the Panel mutation boundary.

## Scope
Implement a Panel-boundary receipt + trust-verb gating slice:
- enforce explicit trust-verb classification (`ASSERT`/`SUGGEST`/`APPLY`) for panel-originated mutation proposals
- require receipt emission fields (verb, authority, basis, outcome, artifact linkage, instance provenance) for admitted APPLY transitions
- ensure SUGGEST remains non-mutating unless promoted through existing governed execution path
- persist compatibility with current outbox/audit envelope contracts

## Source Anchors
- `docs/plans/V60_COGNITIVE_SUPPORT_PRIORITIES.md` :: Priority 3 — Receipts and SUGGEST/APPLY gating before Chat widens
- `docs/CONCEPTS/TRUST_SEMANTICS_CONTRACT.md`
- `docs/CONCEPTS/RECEIPT_TRACE_ACCOUNTABILITY_CONTRACT.md`
- `docs/INTERACTION_SURFACES_AND_AUTHORITY/STATE_EXECUTION_AUTHORITY_REMAINS_GATED.md`
- `docs/SEPARATING_PERSISTENCE_SURFACES/DISTINGUISH_MIRROR_RECEIPT_TRACE.md`

## Constraints
- Keep execution authority unchanged (no direct Chat mutation in this slice).
- Preserve append-only receipt posture.
- Do not collapse receipt and trace semantics.
- Backward-compatible event envelope evolution only.

## Acceptance Criteria
- [ ] Panel mutation pipeline enforces trust-verb classification for relevant proposal paths.  
  Verify: `tests/panel/test_trust_verb_gating.py::test_panel_proposal_requires_trust_verb_classification`.
- [ ] APPLY transitions emit receipt records with required accountability fields.  
  Verify: `tests/events/test_receipt_fields.py::test_apply_receipt_contains_required_fields`.
- [ ] SUGGEST proposals do not mutate without governed promotion/admission path.  
  Verify: `tests/panel/test_trust_verb_gating.py::test_suggest_does_not_mutate_without_admission`.
- [ ] Contract docs reflect the implemented gating + receipt field guarantees.  
  Verify: doc writeback at `docs/contracts/A2A_CONTRACT_AND_TRACE.md` and `docs/STATUS.md`.

## Out of Scope
- Chat mutation authority rollout.
- Receipt storage backend redesign.
- Multi-user authority semantics.

## Suggested Validation
- `PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 pytest -q tests/panel/test_trust_verb_gating.py`
- `PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 pytest -q tests/events/test_receipt_fields.py`

## Source Docs
- `docs/plans/V60_COGNITIVE_SUPPORT_PRIORITIES.md`
- `docs/INTERACTION_SURFACES_AND_AUTHORITY/README.md`
- `docs/SEPARATING_PERSISTENCE_SURFACES/README.md`

### PR DESCRIPTION
**Pull request by RasmusTho**
Fixes #570
Fixes #571
Fixes #572

## Summary
Implements three high-priority ready slices in one bounded runtime batch:
- sync-latency measurement-mode allowlist path and harness verification support (#570)
- derived salience/staleness optional retrieval signal seam (#571)
- panel trust-verb gating and APPLY receipt accountability fields (#572)

## Validation
- `PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 .venv/bin/pytest -q tests/settings/test_panel_watcher_config_validation.py -k summary_action`
- `PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 .venv/bin/pytest -q tests/cli/test_sync_latency_harness.py`
- `PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 .venv/bin/pytest -q tests/retrieval/test_salience_signals.py`
- `PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 .venv/bin/pytest -q tests/retrieval/test_retrieval_capability.py -k signal`
- `PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 .venv/bin/pytest -q tests/panel/test_trust_verb_gating.py`
- `PYTEST_DISABLE_PLUGIN_AUTOLOAD=1 .venv/bin/pytest -q tests/events/test_receipt_fields.py`
- `.venv/bin/python -m app.cli sync-latency-harness --help`

### CODE REVIEW

**Comment by RasmusTho** [comment]
Addressed both unresolved Codex review findings from this PR in follow-up PR #582: https://github.com/RasmusTho/agentic-pkm-mvp/pull/582\n\nCovered fixes:\n1) Catch remote descriptor-list failures and fall back to local registry descriptors.\n2) Re-resolve local descriptor before remote-error fallback execution.

File: app/orchestrator/mcp_tool_provider.py [hunk 1]
```diff
@@ -1,25 +1,47 @@
...
     """Registry-backed tool provider that executes via existing executor paths."""
 
-    def list_descriptors(self) -> Dict[str, ToolDescriptor]:
+    remote_provider: RemoteMCPProvider | None = None
+
+    def list_descriptors(self, tool_settings: Mapping[str, Any] | None = None) -> Dict[str, ToolDescriptor]:
         descriptors = _load_registry_descriptors()
+        if _remote_multiplex_enabled(tool_settings) and self.remote_provider is not None:
+            remote = dict(self.remote_provider.list_descriptors())
+            descriptors.update(remote)
```
[the remark this answers is not in the document]
Comment (RasmusTho) [reply app/orchestrator/mcp_tool_provider.py:39]: Addressed in 3c671da. Execution now resolves descriptors through `_resolve_descriptor_for_execution`, which catches remote descriptor-list failures and flips route to `local_registry` with reason `remote_descriptor_list_error` before invocation.

File: app/orchestrator/mcp_tool_provider.py [hunk 2]
```diff
@@ -46,11 +69,71 @@ def execute_tool_call(
...
+                    step_id=step_id,
+                    description=description,
+                )
+                return dict(response.get("result") if isinstance(response, Mapping) and "result" in response else response)
+            except Exception:
+                route["provider_route"] = "local_registry"
+                route["provider_route_reason"] = "remote_provider_error"
+                route["provider_route_attempted"] = "remote_multiplex"
+
+        return executor._invoke_tool(descriptor, call_args, context, timeout_value)  # noqa: SLF001
```
[the remark this answers is not in the document]
Comment (RasmusTho) [reply app/orchestrator/mcp_tool_provider.py:135]: Addressed in 3c671da. On remote execution error we now re-resolve from local registry and fail with `tool_unavailable` if no local descriptor exists, rather than invoking with a stale/remote descriptor. Added focused tests for both paths in `tests/tools/test_mcp_tool_provider.py`.

**Comment by RasmusTho** [comment]
Follow-up fix for the two unresolved review findings is now in PR #584 (`codex/pr581-review-fix-followup`). It hardens remote descriptor-list failure handling, re-resolves local descriptors on remote-error fallback, and adds explicit tool_unavailable behavior when no local fallback exists.
